editor/components/NotesCard.tsx

```diff
@@ -58,14 +57,11 @@ export default function NotesCard<T extends Record<string, unknown>>({
         rowActionMutation={updateMutation}
         noRowsMessage="No notes found"
         isValidating={false}
-        header={
-          <div className="d-flex justify-content-between">
-            <Card.Title>Notes</Card.Title>
-            <AddNoteButton onClick={handleOpenModal} />
-          </div>
-        }
+        header="Notes"
+        headerButton={<AddNoteButton onClick={handleOpenModal} />}
```

Comment:
separating out optional header buttons into its own prop. this makes it easier to render the column vis picker and any optional header buttons as a group justified to the right of the card header

Reply:
Hmmm imo i think the way things are named now is okay/less confusing bc the only table that uses `headerButton` rn is the Notes table for the add note button. I think if we expand more tables in the future to needing additional header components and they arent buttons we could revisit this, but rn i think the way it is describes whats going on more than headerLeft/headerRight.

alternatively we could maybe update to:
header=> `headerTitle` and headerButton => `additionalHeaderComponent`? 

the EMS Patient Care card is the only one that makes this naming a lil weird bc what would be the `headerTitle` also includes the Beta and info text, but maybe thats fine?

Reply:
i like this idea!